xml/html_title.xml

```diff
@@ -3255,6 +3255,14 @@
     <param pos="0" name="service.product" value="Security Scanner"/>
   </fingerprint>
 
+  <fingerprint pattern="Jellyseerr$">
```

Comment:
Is there a reason for not making the fingerprint more strict to the beginning of the title? This will match anything that ends with `Jellyseerr`.

Reply:
Because of `<example>Sign In - Jellyseerr</example>`

Reply:
Could we modify the fingerprint regex to match anchored to the beginning as well?